test/support/test_public_website.py

```diff
@@ -23,39 +23,138 @@ def __getitem__(self, item):
 
 class TestPublicWebsite(TestCase):
 
-    def test_reset_password_url(self):
-
-        config = MockConfig()
+    def test_reset_password_url_excludes_port_80_when_scheme_is_http(self):
```

Comment:
This is really 3 tests in 1 splitting them up would probably make it more readable